Domains/Smartmetering/smartmeteringwebservices/ClearAlarmRegister.md

```diff
@@ -0,0 +1,15 @@
+## ClearAlarmRegister request
+
+### Description
+ClearAlarmRegister is a request to set the OSGP device communication settings for a specific device. The request needs the DeviceIdentification.
```

Comment:
This looks like a copy-paste error, ClearAlarmRegister is not about setting device communication settings.